### PR TITLE
Fix wrong label key in scheduling example

### PR DESCRIPTION
**Pull request by deRemo**
See the discrepancy here (first pod spec example): https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Fix wrong label key in scheduling example

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 1]
```diff
@@ -124,8 +124,8 @@ For example, consider the following Pod spec:
 
 In this example, the following rules apply:
 
-  * The node *must* have a label with the key `kubernetes.io/e2e-az-name` and
-    the value is either `e2e-az1` or `e2e-az2`.
+  * The node *must* have a label with the key `kubernetes.io/os` and
+    the value is either `linux` or `macOs`.
```
[the remark this answers is not in the document]
Comment (deRemo) [reply content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:128]: Yeah, I wanted to fix the discrepancy without actually changing the original author's context, but you are absolutely right! Let's do it with zones then

File: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 2]
```diff
@@ -124,8 +124,8 @@ For example, consider the following Pod spec:
...
   * The node *preferably* has a label with the key `another-node-label-key` and
     the value `another-node-label-value`.
```
[the remark this answers is not in the document]
Comment (deRemo) [reply content/en/docs/concepts/scheduling-eviction/assign-pod-node.md:130]: still matches

File: content/en/examples/pods/pod-with-affinity-anti-affinity.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@ spec:
             operator: In
             values:
             - linux
+            - macOs
```
[the remark this answers is not in the document]
Comment (deRemo) [reply content/en/examples/pods/pod-with-affinity-anti-affinity.yaml:15]: (Replied in the 3rd conversation)

**Comment by bradtopol** [comment]
Assigning @sftim since he is already engaged on this.
/assign @sftim

**Comment by deRemo** [comment]
@sftim ping! Just in case you forgot

File: content/en/examples/pods/pod-with-affinity-anti-affinity.yaml [hunk 4]
```diff
@@ -29,4 +30,4 @@ spec:
             - key-2
   containers:
   - name: with-node-affinity
-    image: k8s.gcr.io/pause:2.0
```
Comment (zacharysarah) [inline content/en/examples/pods/pod-with-affinity-anti-affinity.yaml:32]: @deRemo 👋🏻 This looks like an unnecessary change. For future PRs, please configure your editor not to enforce newlines.

**Comment by zacharysarah** [comment]
It looks like the author addressed [prior feedback](https://github.com/kubernetes/website/pull/32324#discussion_r838581511) from @sftim, so I'm willing to move this forward. That said, I'd really like a sanity before doing so.

I'm approving this PR but will leave the LGTM to another reviewer.

/approve

File: content/en/examples/pods/pod-with-affinity-anti-affinity.yaml [hunk 5]
```diff
@@ -8,10 +8,11 @@ spec:
       requiredDuringSchedulingIgnoredDuringExecution:
         nodeSelectorTerms:
         - matchExpressions:
-          - key: kubernetes.io/os
```
Comment (kbhawkey) [inline content/en/examples/pods/pod-with-affinity-anti-affinity.yaml:11]: Hello @deRemo . Just checking if the `pods/pod-with-affinity-anti-affinity.yaml` file was updated by mistake? The content page (`assign-pod-node.md`) refers to the example file, `pods/pod-with-node-affinity.yaml`?

**Pull request merged**